### ISSUE TITLE
Support for configuring non-ascii characters to be invalid

### ISSUE THREAD

**Issue by frodeto** [RohanNagar/jmail#167]
**Please describe the feature that you are requesting**

It seems to me that email servers / SMTP servers often (normally?) do not support non-ascii characters in local part or domain, even in countries where it is common with non-ascii characters in names (such as æ, ø, å in Norway).
It would be really nice with a feature for configuring non-ascii characters not to be valid.
E. g., "jørn@test.com" should fail.

**Additional context**

**Comment by RohanNagar** [comment]
Hi @frodeto, thanks for opening this issue! Somewhat related to #149, but this seems like a good enhancement that would be straightforward to implement. I'm going to try and get this added in the next week.

**Comment by RohanNagar** [comment]
This feature is now available in v1.6.0! Example:

```java
EmailValidator validator = JMail.strictValidator().requireAscii();

validator.isValid("jørn@test.com"); // returns false
```
